src/fetchAPI.ts

```diff
@@ -661,4 +661,28 @@ export async function getAtCommandPreview(query: string): Promise<ChatContextFil
       );
 
       return jsonMessages;
-}
\ No newline at end of file
+}
+
+export async function get_statistic_data(): Promise<any> {
+    let url = rust_url("/v1/get-dashboard-plots");
+
+    if (!url) {
+      return Promise.reject("get-dashboard-plots doesn't work");
+    }
+
+    let req = new fetchH2.Request(url, {
+      method: "GET",
+      redirect: "follow",
+      cache: "no-cache",
+      referrer: "no-referrer",
+    });
+
+    let resp = await fetchH2.fetch(req);
+    if (resp.status !== 200) {
+        console.log(["get_dashboard_plots http status", resp.status]);
+        return Promise.reject("get_dashboard_plots");
```

Comment:
I had an issue with a 500 response in the other PR, but didn't see a message :/
is it being sent through the event bus?